packages/site/src/pages/api/data/data.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
```

Comment:
You could have modified the sitemap plugin that we already have to do this I think.  We don't need to do it this way